vocabularies/Analytics.xml

```diff
@@ -39,14 +39,38 @@
       </Annotation>
 
       <Term Name="Dimension" Type="Core.Tag" BaseTerm="Aggregation.Groupable" Nullable="false" DefaultValue="true" AppliesTo="Property">
+        <Annotation Term="Core.Revisions">
+          <Collection>
+            <Record>
+              <PropertyValue Property="Kind" EnumMember="Core.RevisionKind/Deprecated"/>
+              <PropertyValue Property="Description" String="Deprecated in favor of [`AnalyticalContext/Dimension`](#AnalyticalContext)"/>
+            </Record>
+          </Collection>
+        </Annotation>
```

Comment:
```suggestion
      <Term Name="Dimension" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Property">
        <Annotation Term="Core.AppliesViaContainer"/>
```
It would be simpler to keep the property-targeted term but add the "applies via container" tag (and remove the deprecated base term?).